docs-chef-io/content/inspec/resources/file.md

```diff
@@ -587,3 +596,11 @@ return `true` if your file has a mode with greater permissions than specified.
       it { should_not be_more_permissive_than('0644') }
       it { should be_more_permissive_than('0000') }
     end
+
+### `be_inherited`
+
+`be_inherited` matcher returns the `Boolean`. It will return `true` if file or foler has inheritance enabled in Windows. This matcher only works for Windows OS.
```

Comment:
```suggestion
`be_inherited` is a boolean matcher which returns `true` if a file or folder has inheritance enabled, otherwise `false`. This matcher only works on Windows systems.
```